internal/gensupport/resumable.go

```diff
@@ -239,3 +224,21 @@ func (rx *ResumableUpload) Upload(ctx context.Context) (resp *http.Response, err
 		return prepareReturn(resp, err)
 	}
 }
+
+// shouldRetry indicates whether an error is retryable for the purposes of this
+// package.
+func shouldRetry(status int, err error) bool {
+	if 500 <= status && status <= 599 {
```

Comment:
This doesn't include connectReset, is that expected?

Reply:
I put that change in a separate commit, can add it in this PR as well if you like.

Reply:
That's fine. Let's keep this change focused just wanted to verify.